definitions-tests/definitions_test.go

```diff
@@ -0,0 +1,74 @@
+package definitions_tests
+
+import (
+	"cuelang.org/go/cue"
+	"cuelang.org/go/cue/cuecontext"
+	"github.com/stretchr/testify/assert"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+const (
+	velaTemplatesDir   = "../vela-templates/definitions/"
+	testDataDir        = "./test-data/"
+	expectedOutputsKey = "outputs"
```

Comment:
We need to check `output` & `outputs`. It would also be better if we can also check `customStatus` & `healthPolicy` here.

Reply:
I have added `expectedKeys = []string{"outputs", "output"}` in `definition_test.go`, so they can check these keys if they are also existing in the `definition`. 
For `customStatus` and `healthPolicy`, there is a `status` field in them, I don't know how to check this filed.

Reply:
```
customStatus := a.LookupPath(cue.ParsePath("status.customStatus"))
```

Reply:
Do I need to compile the `customStatus ` value, then evaluate it with test data?